src/schema/v2/artist/createArtistMutation.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  GraphQLBoolean,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { GraphQLObjectType } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { snakeCase } from "lodash"
+import { ResolverContext } from "types/graphql"
+import { ArtistType } from "./index"
+
+interface CreateArtistMutationInput {
+  birthday: string
+  deathday: string
+  displayName: string
+  first: string
+  isPersonalArtist: boolean
+  last: string
+  middle: string
+  nationality: string
```

Comment:
**Suggestion**: I got a bit confused when I first saw `first`, `middle`, and `last`, should we append `Name` to them to make them clear? assuming that they're related to the first, middle, and last name 😅

Reply:
good suggestion - I agree regardless of the fact that this is how they're called in Gravity.

Reply:
But We might not need any of them given that we don't have them on the [frontend](https://www.figma.com/file/ARjzNu0OVYVcpeGjRgXkiY/Artists-Collected?type=design&node-id=1-14841&t=mi08LzkohoTzPWrg-0)  and we only have the full name

Reply:
I would probably stick to matching Gravity's arguments, in terms of introducing new vocabulary for a domain concept.

But, a property named `first` _is_ a little bit vague, probably `firstName` would be better, and that's not an important domain concept, it's just a field on artist (where 'artist' is the important domain concept).

So, I would feel free to change it here to make it more clear, just be careful about introducing new terminology for domain-level things.